client/src/api/axios.ts

```diff
@@ -1,17 +1,57 @@
-import axios, { AxiosInstance } from 'axios';
-
+import axios, { AxiosInstance } from 'axios'
+import md5 from 'md5'
 const AuthClient: AxiosInstance = axios.create({
-  baseURL: import.meta.env.VITE_APP_ENDPOINT, 
-  timeout: 1000,
+  baseURL: 'https://server.gent02.dev.grid.tf/api',
   headers: {
     'Content-Type': 'application/json',
-    'Authorization': 'Bearer ' + localStorage.getItem("token"),
+    Authorization: 'Bearer ' + localStorage.getItem('token'),
   },
-});
+})
 
 const BaseClient: AxiosInstance = axios.create({
-  baseURL: import.meta.env.VITE_APP_ENDPOINT,
-  timeout: 1000,
-});
+  baseURL: 'https://server.gent02.dev.grid.tf/api',
+})
+
+async function SignUp (newUser:any) {
+  try {
+    await BaseClient.post('/auth/signup/', newUser)
+  } catch (error) {
+    console.error(error)
+    throw error
+  }
+}
+
+async function SignUpInvitation (passwords:any) {
+  try {
+    await BaseClient.put('/members/set_password/', passwords)
+  } catch (error) {
+    console.error(error)
+    throw error
+  }
+}
+
+async function LogInUser (userInfo:any) {
+  try {
+    const response = await BaseClient.post('/auth/login/', userInfo)
+    const token = response.data.access_token
+    localStorage.setItem('token', token)
+    localStorage.setItem('password', md5(userInfo.password))

```

Comment:
```suggestion
    localStorage.setItem('TEST_TRACKER_ACCESS_TOKEN', token)
    localStorage.setItem('TTPHASH', md5(userInfo.password))
```
TTPHASH is `TestTrackerPasswordHash`
You should also save the `Refresh Token`